src/ui/MLFacet/MLFacet.ts

```diff
@@ -293,12 +295,40 @@ export class MLFacet extends Component {
    */
   public reset() {
     this.ensureDom();
+    this.logger.info('Deselect all values');
     this.values.clearAll();
     this.values.render();
     this.updateAppearance();
     this.updateQueryStateModel();
   }
 
+  /**
+   * Collapses or expands the facet depending on it's current state.
+   */
+  public toggleCollapse() {
+    this.isCollapsed ? this.expand() : this.collapse();
+  }
+
+  /**
+   * Expands the facet. Displays all values.
```

Comment:
About _displays all values_, would _displays all currently fetched values_ be more accurate?

If so, here's a rephrased suggestion:

> Expands the facet, displaying all of its currently fetched values.